<!DOCTYPE HTML>
<html>
<head>
<meta charset="UTF-8">
<title>Proteases</title>
<link href="styles/main.css" rel="stylesheet" type="text/css">
</head>

<body>
<div id="wrapper">
  <header id="top">
    
    <img src="CULogo with text copy.png" width="353" height="120" alt="CULogo">
    <nav id="mainnav">
      <ul>
        <li><a href="index.html">HOME</a></li>
        <li><a href="projects.html">PROJECTS</a></li>
        <li><a href="publications.html">PUBLICATIONS</a></li>
        <li><a href="people.html">PEOPLE</a></li>
        <li><a href="support.html">SUPPORT</a></li>
        <li><a href="contact.html">CONTACT</a></li>
      </ul>
    </nav>
    <p data-gwd-node="true" data-gwd-uuid="yb2l2qbzlazc"> </p>
    
    <article id="Titlebar">
      <article id="SubTitle">
       	<p id="spacer"> </p>
        <h1>CELL DEATH PROTEASES</h1>
      </article>
      <article id="Titledesc">
        <h2>Characterization of proteases involved in cell death pathways</h2>
      	<p data-gwd-node="true" data-gwd-uuid="yb2l2qbzlazc"> </p>
      </article>
    </article>
  </header>
  </div>
<article id=maintext>
  	<p data-gwd-node="true" data-gwd-uuid="yb2l2qbzlazc"> </p>
    <p>1) <strong>Identification and characterization of downstream pathways of the CED-3 cell death protease</strong></p>
    <p>Analysis of downstream pathways of important enzymatic biomolecules (e.g., kinases and proteases) that have multiple substrates has always been a difficult challenge. Targets of these enzymes could be difficult to identify through conventional genetic screens, since elimination of one of the multiple targets of an enzyme may fail to cause any visible phenotype that can be scored in genetic screens.  Enzymatic targets could also be difficult to identify through commonly used biochemical approaches such as protein interaction screens that require stable interaction rarely seen between an enzyme and a substrate.  In fact, to date there have been no good methods available that can systematically and effectively address the problem of substrate identification.</p>
    <p> </p>
    <img src="Proteases1.png" width="350" height="418" alt="proteases1">
<p> </p>
    <p>During apoptosis, a family of aspartate-specific cysteine proteases named caspases are activated proteolytically to execute the apoptotic program.  Little is known about the identities of their in vivo targets or their downstream pathways that mediate the killing functions.  We have developed a novel, GFP-based genetic screen to identify downstream targets of the <em>C. elegans</em> caspase, CED-3.  In this screen, the green fluorescent protein (GFP) and a constitutively activated version of CED-3 were co-expressed in <em>C. elegans</em> mechanosensory neurons and GFP was used as a sensitive cell existing marker to isolate mutations that either partially suppress or delay ectopic neuronal deaths caused by the activated CED-3 (Figure 1).  These CED-3 protease suppressors (we named these <em>cps</em>mutations) likely affect genes that act downstream of, or in parallel to, ced-3 to mediate various cell-killing processes.  We have screened approximately 150,000 <em>C. elegans</em> haploid genomes and isolated more than 80 cps mutations.   Phenotypic analyses of these cps mutants suggest that these mutations not only suppress CED-3-induced ectopic neuronal deaths but also affect normal programmed cell death in <em>C. elegans</em>, indicating that they are true cell-death mutants. Genetic analyses of these cps mutations indicate that they affect at least fourteen new cell death genes (<em>cps-1</em> to <em>cps-14</em>) and one previously identified gene (<em>ced-1</em>), which acts downstream of <em>ced-3</em> to remove apoptotic cell corpses.  Phenotypic analyses of these cps mutants suggest that they can be categorized into three major groups: 1) mutations that weakly suppress cell deaths in <em>C. elegans</em> (<em>cps-1</em> and<em> cps-2</em>), 2) mutations that delay the normal progression of apoptosis (<em>cps-3, 4, 5, 6, 7, 8, 10,</em> and <em>11</em>), and 3) mutations that may cause defect in the removal of cell corpses (<em>cps-9, 12, 13, 14</em>).  These 14 <em>cps</em> genes may encode important components that act downstream of the CED-3 protease to regulate or execute different aspects of the cell disassembly and removal process (Figure 2).  Indeed, using the TUNEL assay (an assay used to detect DNA breaks generated during apoptosis), we found that<em> cps-3, 4, 6, 7, 8, 10, 11, 13</em> and <em>14 </em>genes are involved in the chromosome fragmentation process, one of the key steps and a hallmark of apoptosis.  In addition, using an annexin V staining assay (used to detect surface-exposed phosphatidylserine), we found that three genes (<em>cps-12, cps-13,</em> and<em> cps-14</em>) affect externalization of phosphatidylserine (PS) in apoptotic cells.  PS normally is restricted to the inner leaflet of the plasma membrane but is exposed or flipped out in apoptotic cells and has been suggested to serve as an engulfment signal for phagocytosis.  How PS is flipped out in dying cells during apoptosis has not been clear and is a topic of intense studies.  Molecular characterization of <em>cps-12, cps-13</em> and <em>cps-14</em> will likely provide important insights towards addressing this fundamental biological question.</p>
    <p> </p>
    <p>We recently cloned the <em>cps-6</em> gene and found that <em>cps-6</em> encodes a homologue of human mitochondrial endonuclease G (endoG).  In collaboration with Dr. Xiaodong Wang's group at the University of Texas Southwestern Medical Center, we showed that the CPS-6 protein localizes to <em>C. elegans</em> mitochondria and possesses an endonuclease activity that is capable of inducing the generation of apoptotic DNA ladders in isolated Hela cell nuclei.  Furthermore, the mouse EndoG can substitute for the functions of cps-6 in <em>C. elegans</em>, suggesting that CPS-6 and EndoG define an evolutionarily conserved DNA degradation pathway.  Our study also demonstrates for the first time that mitochondria is important for apoptosis in invertebrates and is a conserved regulator of apoptosis (<a href="http://www.nature.com/cgi-taf/DynaPage.taf?file=/nature/journal/v412/n6842/abs/412090a0_fs.html">Abstract</a> and PDF).</p>
    <p> </p>
    <p>2) <strong>Characterization of the <em>C. elegans</em> mitochondrial cell death pathways</strong></p>
    <p>The findings that apoptotic regulators such as cytochrome c, endoG, and apoptosis-inducing-factor (AIF) are released from mitochondria to mediate different aspects of apoptosis indicate that mitochondria plays an important role in mammalian apoptosis.  A family of proteins containing a unique Bcl-2 homology 3 motif (BH3) are involved in releasing these mitochondrial apoptogenic factors.  The finding that <em>cps-6</em> encodes a mitochondrial endonuclease prompted us to examine whether there are additional mitochondrial proteins involved in <em>C.elegans</em> apoptosis.  We characterized the <em>C. elegans</em> homologue of AIF, a human mitochondrial oxidoreductase that is released from mitochondria during apoptosis to induce chromosome condensation and fragmentation.  Intriguingly, the oxidoreductase activity of AIF is dispensable for its apoptogenic functions and AIF does not possess a nuclease activity, raising a major question of how AIF induces apoptosis.  We cloned the worm AIF homologue, <em>wah-1</em>, and used RNA interference (RNAi) to study its functions in <em>C. elegans</em> apoptosis.  We found that reduction of the <em>wah-1</em> activity in <em>C.elegans</em> delays the normal progression of apoptosis, results in accumulation of TUNEL-positive DNA breaks, and enhances the defects of other cell death mutants, indicating that <em>C. elegans</em> AIF does play an important role in regulating <em>C. elegans</em> apoptosis.  Interestingly, <em>wah-1</em>(RNAi) results in cell death phenotypes that are similar to those of the cps-6 mutant and fails to enhance the cps-6 cell death defects, indicating that <em>wah-1</em> and <em>cps-6</em> function in the same cell death pathway.  Indeed, we found that WAH-1, which also localizes to <em>C. elegans</em> mitochondria, can associate and cooperate with CPS-6 in vitro to promote DNA degradation.  In vivo, co-expression of <em>wah-1</em> and <em>cps-6</em> can synergistically induce cell killing.  These findings indicate that CPS-6/EndoG is likely a target or an effector that WAH-1/AIF interacts with to induce chromosome fragmentation during apoptosis and that CSP-6/EndoG and WAH-1/AIF define a single evolutionarily conserved cell death pathway initiated from mitochondria.  Importantly, WAH-1 can be released from mitochondria by EGL-1, a C. elegans BH3-domain containing cell death activator, in a manner similar to the release of cytochrome c or EndoG from mitochondria by mammalian BH3-domain containing proteins such as Bid or Bim.  However, this EGL-1-induced release of WAH-1 is dependent on the activity of the CED-3 protease, indicating that worm AIF functions in a caspase-dependent manner.  These observations strongly suggest that the mitochondrial cell death pathway is conserved between nematodes and humans (<a href="http://www.sciencemag.org/cgi/content/abstract/298/5598/1587">Abstract</a> and <a href="PDF Files/WAH-1.pdf">PDF</a>).</p>
    <p> </p>
    <img src="proteases2.png" width="350" height="281" alt="proteases2">
<p> </p>
    <p>We are currently focusing on identifying additional mitochondrial factors that are important for the regulation and execution of programmed cell death in <em>C. elegans</em> using both genetic and biochemical approaches.  We are particularly interested in understanding how mitochondrial apoptogenic factors are released from mitochondria to affect various aspects of apoptosis.  We hope to identify most components functioning in <em>C. elegans</em> mitochondrial cell death pathways.</p>
    <p> </p>
    <p><strong>3) Functional genomic analysis of the apoptotic DNA degradation process in <em>C. elegans</em></strong></p>
    <p>The observations that multiple genes (<em>cps-3, 4, 6, 7, 8, 10, 11, 13, 14</em> and <em>wah-1</em>) are involved in the apoptotic DNA degradation process in<em> C. elegans</em> indicate that this is likely a rather complicated and tightly regulated process.  To identify all the nucleases involved in <em>C. elegans</em> apoptosis, we conducted a candidate-based, genome-wide RNAi screen to systematically search for genes important for apoptotic DNA degradation in <em>C. elegans</em>.  We used RNAi to individually inhibit the expression of 77 <em>C. elegans </em>genes that encode nucleases or nuclease-related proteins and have identified nine candidate apoptotic nucleases, including two previously known apoptotic nucleases (CPS-6 and NUC-1).  We named these new cell death-related nucleases as CRN nucleases.  Molecular genetic analyses of these crn genes indicate that these nine apoptotic nucleases comprise at least two independent pathways that contribute to cell killing by degrading chromosomal DNA, with <em>cps-6, crn-1, crn-4, crn-5, </em>and <em>crn-7 </em>acting together in one pathway and <em>crn-2/crn-3</em> in another pathway (Figure 2).  <em>nuc-1</em> and <em>crn-6</em> appear to act at later stages of apoptotic DNA degradation.  Interestingly, several crn genes have human homologues that play important roles in RNA processing and splicing, protein folding, DNA replication and damage repair, suggesting that these CRN proteins may play dual roles in both cell survival and cell death.  The identification of seven crn genes will allow systematic deciphering of the mechanisms of apoptotic DNA degradation, which remain a poorly understood biological process (<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&list_uids=12718884&dopt=Abstract">Abstract</a> and <a href="PDF Files/TUNEL Screen.pdf">PDF</a>).</p>
    <p> </p>
    <p>As a first step towards the understanding of how apoptotic nucleases interact to promote apoptotic DNA degradation, we initiated biochemical and mechanistic studies of CRN-1, a homologue of the human Fen-1 endonuclease that plays important roles in DNA replication and repair.  We found that CRN-1 localizes to nuclei and can associate and cooperate with CPS-6 to promote stepwise DNA fragmentation, utilizing the endonuclease activity of CPS-6 and both the 5'-3' exonuclease activity and a novel gap-dependent endonuclease activity of CRN-1.  Our results suggest that CRN-1/FEN-1 may play a critical role in switching the state of cells from DNA replication/repair to DNA degradation during apoptosis (<a href="http://emboj.oupjournals.org/cgi/content/abstract/22/13/3451">Abstract</a> and  <a href="PDF Files/CRN-1.pdf">PDF</a>).</p>
    <p> </p>
    <p><strong>4) Identification and characterization of genes involved in exposure or recognition of 'eat-me' signals during removal of apoptotic cell corpses</strong></p>
    <p>Phagocytosis of apoptotic cells is an integral part of cell death execution and an important event in tissue remodeling, suppression of inflammation, and regulation of immune responses. During apoptosis, 'eat-me' or engulfment signals are exposed or released from the dying cells to trigger the phagocytic events by neighboring cells or macrophages.  Very little is known about what these 'eat-me' signals are or how they are exposed or released from the dying cells.  One of the candidate 'eat-me' signals is phosphatidylserine (PS), which normally is restricted to the inner leaflet of the plasma membrane but is exposed on the surface of apoptotic cells.  It is unclear what regulates the externalization of PS in apoptotic cells and how phagocytes recognize PS and subsequently initiate the phagocytic events.  From our cps screens, we have identified three genes (<em>cps-12, cps-13</em> and<em> cps-14</em>) that affect the externalization of PS in apoptotic cells. Molecular characterization of these three genes will provide important insights on how PS externalization is regulated and executed in apoptotic cells.</p>
    <p> </p>
    <p>Recently, a putative phosphatidylserine receptor (PSR) was identified and proposed to mediate PS recognition and phagocytosis.  Interestingly, <em>C. elegans</em> contains a gene (F29B9.4) that shares 56% sequence identity with human PSR.  To investigate whether the<em> C. elegans </em>PSR homologue (named <em>psr-1</em>) affects cell corpse engulfment, we isolated a deletion allele (<em>tm469</em>) in the <em>psr-1</em> locus and analyzed its mutant phenotypes.  We found that the <em>psr-1(tm469) </em>mutation does result in a defect in cell corpse engulfment.  Interestingly, <em>psr-1</em> appears to act in the same cell corpse engulfment pathway as <em>ced-2, ced-5, ced-10 </em>and <em>ced-12</em>.  Genetic bypass experiments indicate that <em>psr-1</em> acts upstream of <em>ced-2, ced-5, ced-10</em> and <em>ced-12</em> to control cell corpse engulfment.  <em>In vitro C. elegans</em> PSR-1 behaves like human PSR and binds preferentially PS or cells with exposed PS.  Interestingly, the intracellular domain of PSR-1 interacts with CED-5 and CED-12, suggesting that PSR-1 may transduce engulfment signal through CED-5 and CED-12.  Our findings suggest that PSR-1 is likely an upstream receptor for the signaling pathway containing CED-2, CED-5, CED-10 and CED-12 proteins and plays an important role in recognizing phosphatidylserine during phagocytosis (<a href="http://www.sciencemag.org/cgi/content/abstract/302/5650/1563">Abstract</a> and <a href="PDF Files/psr-1.pdf">PDF</a>).</p>
    <p> </p>
    <p>&nbsp;</p>
    <p><strong>Related Publications:</strong></p>
    <p> </p>
    <p>Parrish, J., Li, L., Klotz, K., Ledwich, D., Wang, X.D., and Xue, D. (2001).  <em>C. elegans</em> mitochondrial endonuclease G is important for apoptosis.  <strong>Nature</strong> 412, 90-94. (<a href="http://www.nature.com/cgi-taf/DynaPage.taf?file=/nature/journal/v412/n6842/abs/412090a0_fs.html">Abstract</a> and PDF). <strong>Nature Reviews Mol. Cell Biol</strong>.</p>
    <p> </p>
    <p>Wang, X.C., Yang, C.L., Cai, J.J., Shi, Y.G., and Xue, D. (2002).  Mechanisms of AIF-mediated apoptotic DNA degradation in <em>Caenorhabditis elegans</em>.  <strong>Science</strong> 298, 1587-1592.  (<a href="http://www.sciencemag.org/cgi/content/abstract/298/5598/1587">Abstract</a> and<a href="file://localhost/PDF%252520Files/WAH-1.pdf">PDF</a>).  <a href="http://www.nature.com/cgi-taf/DynaPage.taf?file=/nrm/journal/v4/n1/full/nrm1010_fs.html">Nature Reviews Molecular Cell Biology</a> (<a href="PDF Files/Nature Reviews-AIF.pdf">PDF</a>)</p>
    <p> </p>
    <p>Parrish, J. and Xue, D. (2003). Functional genomic analysis of apoptotic DNA degradation in <em>C. elegans</em>.  <strong>Mol. Cell </strong>11, 987-996.  (<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&list_uids=12718884&dopt=Abstract">Abstract</a> and <a href="PDF Files/TUNEL Screen.pdf">PDF</a>)</p>
    <p> </p>
    <p>Parrish, J., Yang, C.L., Shen, B.H., and Xue, D. (2003). CRN-1, a<em> Caenorhabditis elegans</em> FEN-1 homologue, cooperates with CPS-6/EndoG to promote apoptoticDNA degradation.  <strong>EMBO</strong>. J. 22, 3451-3460. (<a href="http://emboj.oupjournals.org/cgi/content/abstract/22/13/3451">Abstract</a> and <a href="PDF Files/CRN-1.pdf">PDF</a>)</p>
    <p> </p>
    <p>Wang, X.C., Wu, Y.C., Fadok, V., Lee, M.C., Gengyo-Ando, K., Cheng, L.C., Ledwich, D., Hsu, P.K., Chen, J.Y., Chou, B.K., Henson, P., Mitani, S., and Xue, D. (2003).  Cell Corpse Engulfment Mediated by<em> C. elegans</em> Phosphatidylserine Receptor Through CED-5 and CED-12. <strong>Science</strong> 302, 1563-1566. (<a href="http://www.sciencemag.org/cgi/content/abstract/302/5650/1563">Abstract</a> and <a href="PDF Files/psr-1.pdf">PDF</a>).  <a href="PDF Files/Perspective-psr-1.pdf">Science Perspectives (PDF)</a></p>
    <p> </p>
    <p>Wang, X.C., Wang, J., Gengyo-Ando, K., Gu, L.C., Sun, C.L., Yang, C.L., Shi, Y., Kobayashi, T., Shi, Y.G., Mitani, S., Xie, X.S., and Xue, D. (2007). "<em>C. elegans</em> mitochondrial factor WAH-1 promotes phosphatidylserine externalization in apoptotic cells through phospholipid scramblase SCRM-1". <strong>Nature Cell Biology</strong> 9, 541-549. (<a href="http://www.nature.com/ncb/journal/v9/n5/abs/ncb1574.html">Abstract</a> and <a href="PDF Files/ncb1574.pdf">PDF</a>). <a href="http://www.nature.com/nrm/journal/v8/n5/full/nrm2176.html">Nature Reviews Molecular Cell Biology</a> (<a href="PDF Files/nrm2176.pdf">PDF</a>)</p>
    <p> </p>
    <p>Darland-Ransom, M., Wang, X.C., Sun, C.L., Mapes, J., Gengyo-Ando, K., Mitani, S. and Xue, D. (2008). Role of <em>C. elegans</em> TAT-1 protein in maintaining plasma membrane phosphatidylserineasymmetry. <strong>Science</strong> 320, 528-531. (<a href="http://www.sciencemag.org/cgi/content/abstract/320/5875/528">Abstract</a> and <a href="PDF Files/tat-1.pdf">PDF</a>).  <a href="PDF Files/Perspective-tat-1.pdf">Science Perspectives (PDF)</a></p>
    <p> </p>
    <p>Breckenridge, D., Kang, B.H., Kokel, D., Mitani, S., Staehelin, A.L., and Xue, D. (2008). <em>Caenorhabditis elegans</em> drp-1 and fis-2 regulate distinct cell death execution pathways downstream of ced-3and independent of ced-9.  <strong>Mol. Cell</strong> 31 586-597. (<a href="http://www.sciencedirect.com/science?_ob=ArticleURL&_udi=B6WSR-4T8B978-C&_user=918210&_rdoc=1&_fmt=&_orig=search&_sort=d&view=c&_acct=C000047944&_version=1&_urlVersion=0&_userid=918210&md5=d2abe6c836f8d6a35c1ee4f22a5ba8e5">Abstract</a> and <a href="PDF Files/drp-1.pdf">PDF</a>)</p>
    <p> </p>
    <p>Breckenridge, D., Kang, B.H., and Xue, D. (2009).  Bcl-2 proteins EGL-1 and CED-9 do not regulate mitochondrial fission or fusion in <em>Caenorhabditis elegans</em>.  <strong>Current Biology </strong>19, 768-773.(<a href="http://www.cell.com/current-biology/abstract/S0960-9822(09)00825-2">Abstract</a> and <a href="PDF Files/CED-9 mito.pdf">PDF</a>)</p>
    <p> </p>
    <p>Hsiao, Y.Y., Nakagawa, A., Shi, Z., Mitani, S., Xue, D. and Yuan, H. S. (2009). Crystal structure of CRN-4: implications for domain function in apoptotic DNA degradation. <strong>Mol. Cell. Biol. </strong>29, 448-457.  (<a href="http://mcb.asm.org/cgi/content/full/29/2/448?view=long&pmid=18981218">Abstract</a> and <a href="PDF Files/CRN-4.pdf">PDF</a>)</p>
    <p> </p>
    <p>Lai, H.J., Lo, S.Z., Kage-Nakadai, E., Mitani, S., and Xue, D. (2009). The roles and acting mechanism of <em>Caenorhabditis elegans</em> DNase II genes in apoptotic DNA degradation and development. <strong>PLoS One</strong> 4, e7348. (<a href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0007348#cor1">Abstract</a> and PDF)</p>
    <p> </p>
    <p>Nakagawa, A.*, Shi, Y.*, Kage-Nakadai, E., Mitani, S., and Xue, D. (2010). Caspase-Dependent Conversion of Dicer Ribonuclease into a Death-Promoting Deoxyribonuclease. <strong>Science</strong> 328, 327-334. *Equal contributions (<a href="http://www.sciencemag.org/cgi/content/short/328/5976/327">Abstract</a> and <a href="http://www.sciencemag.org/cgi/rapidpdf/science.1182374?ijkey=UFAxQ5diEjwoA&keytype=ref&siteid=sci">PDF</a>). Research Article featured on the <a href="http://www.sciencemag.org/content/vol328/issue5976/cover.dtl">cover</a> of Science and <a href="PDF Files/Perspectives-dcr-1.pdf">Science Perspectives</a>, <a href="PDF Files/Dicer DCell Preview.pdf">Developmental Cell Preview</a>, <a href="PDF Files/nsmb_highlight.pdf">Highlights in Nature Structural and Molecular Biology</a>, <a href="PDF Files/nrm2908.pdf">Nature Reviews Molecular Cell Biology</a>, <a href="PDF Files/DMM_highlight.pdf">Disease Models and Mechanisms</a>, and <a href="PDF Files/Dicer-F1000.pdf">Faculty of 1000</a>    </p>
    <p> </p>
    <p>Wang, X.C., Li W., Zhao, D.F., Liu, B., Shi, Y., Chen, B.H., Yang, H.W., Guo, P.F., Geng, X., Shang, Z.H., Peden, E., Kage-Nakadai, E., Mitani, S., and Xue, D. (2010). <em>C. elegans</em> transthyretin-like protein TTR-52 mediates recognition of apoptotic cells by the CED-1 phagocyte receptor.  <strong>Nature Cell Biology</strong>, in press. (<a href="http://www.nature.com/ncb/journal/vaop/ncurrent/abs/ncb2068.html">Abstract</a> and <a href="PDF Files/ncb2068.pdf">PDF</a>) <a href="PDF Files/N&VTTR-52.pdf">Nature Cell Biology News and Views</a></p>
    <p> </p>
    <p>Lin, J.L., Nakagawa, A., Lin, C.L., Hsiao, Y.Y., Yang, W.Z., Wang, Y.T., Doudeva, L.G., Skeen-Gaar, R.R., Xue, D., and Yuan, H.S. (2012). Structural insights into apoptotic DNA degradation by CED-3 Protease Suppressor-6 (CPS-6) from <em>Caenorhabditis elegans</em>. <strong>Journal of Biological Chemistry</strong> 287:7110-7120 (<a href="http://www.jbc.org/content/early/2012/01/05/jbc.M111.316075.abstract?sid=2d7025d1-69b3-4453-9897-d4d9f1feae62">Abstract</a> and <a href="http://www.jbc.org/content/early/2012/01/05/jbc.M111.316075.full.pdf+html">PDF</a>).</p>
    <p> </p>
    <p>Mapes, J., Chen, Y.Z., Kim, A., Mitani, S., Kang, B.H., and Xue, D. (2012). CED-1, CED-7, and TTR-52 act in a pathway to regulate exoplasmic phosphatidylserine expression on apoptotic and phagocytic cells. <strong>Current Biology</strong> 22, 1267-1275. (<a href="http://www.cell.com/current-biology/abstract/S0960-9822(12)00643-4">Abstract</a> and <a href="PDF Files/exPS mechanisms.pdf">PDF</a>). Faculty of 1000</p>
    <p> </p>
    <p>Chen, Y.Z., Mapes, J., Lee, E.S. and Xue, D. (2013). Caspase-mediated activation of <em>Caenorhabditis elegans</em> CED-8 promotes apoptosis and PS externalization. <strong>Nature Communications</strong> 4:2726doi: 10.1038/ncomms3726. (<a href="http://www.nature.com/ncomms/2013/131114/ncomms3726/full/ncomms3726.html">Abstract</a> and <a href="PDF Files/Chen.pdf">PDF</a>)</p>
    <p> </p>
    <p>Ge, X., Zhao, X., Nakagawa, A., Gong, X., Skeen-Gaar, R., Shi, Y., Gong, H.P., Wang, X.Q., and Xue, D. (2013). A novel mechanism underlies caspase-dependent conversion of the dicerribonuclease into a deoxyribonuclease during apoptosis. <strong>Cell Research</strong> 24: 218-232 (<a href="http://www.nature.com/cr/journal/vaop/ncurrent/abs/cr2013160a.html">Abstract</a> and <a href="PDF Files/dicer_mechanism.pdf">PDF</a>).</p>
    <p> </p>
    <p>Nakagawa, A., Sullivan, K., and Xue, D. (2014). Caspase-activated phosphoinositide binding by CNT-1 promotes apoptosis by inhibiting the AKT pathway. <strong> Nature Structural &amp; Molecular Biology </strong> 21, 1082-1090. (<a href="http://www.nature.com/nsmb/journal/vaop/ncurrent/full/nsmb.2915.html">Abstract</a> and PDF).</p>
    <p data-gwd-node="true" data-gwd-uuid="yb2l2qbzlazc"> </p>
    <p>Yang, H.W.*, Chen, Y.Z.*, Zhang, Y.*, Wang, X.H., Zhao, X., Godfroy, J.I., Liang, L., Zhang, M., Zhang, T.Y., Yuan, Q., Royal, M.A., Driscoll, M.D., Xia, N.S., Yin, H., and Xue, D.  A lysine-rich in the phosphatidylserine receptor PSR-1 mediates recognition and removal of apoptotic cells. <strong> Nature Communications </strong>, 6: 5717 doi: 10.1038/ncomms6717. *These authors contributed equally to this work (<a href="http://www.nature.com/ncomms/2015/150107/ncomms6717/full/ncomms6717.html">Abstract</a> and PDF). </p>
    <p data-gwd-node="true" data-gwd-uuid="yb2l2qbzlazc"> </p>
    <p>Weaver, B.P., Zabinsky, R., Weaver, Y.M., Lee, E.S., Xue. D., and Han, M. (2014). CED-3 caspase acts with miRNAs to regulate non-apoptotic gene expression dynamics for robust development in C. elegans. <strong> eLife </strong> doi: 10.7554/eLife.04265. (<a href="http://www.elifesciences.org/content/early/2014/12/02/eLife.04265">Abstract</a> and <a href="PDF Files/CED-3 caspase acts with miRNAs to regulate non-apoptotic gene expression dynamics for robust development in C. elegans.pdf">PDF</a>).</p>
    <p>&nbsp;</p>
    <p>Neumann, B., Coakley, S., Giordano-Santini, R., Linton, C., Lee, E.S., Nakagawa, A., Xue, D., and Hilliard, M.A. (2015).  EFF-1-mediated regenerative axonal fusion requires components of the apoptotic pathway. <strong> Nature </strong> 517, 219–222 (<a href="http://www.nature.com/nature/journal/v517/n7533/full/nature14102.html">Abstract</a> and <a href="PDF Files/nature14102.pdf">PDF</a>).</p>
    <p>&nbsp;</p>
    <p>Sullivan, K.*, Nakagawa, A.*, Xue, D.#, and Espinosa, J.M.# (2015). Human ACAP2 is a homolog of <em>C. elegans</em> CNT-1 that promotes apoptosis in cancer cells. <strong> Cell Cycle </strong>, in press. *Equal contribution. #Co-corresponding authors. (<a href="http://www.tandfonline.com/doi/abs/10.1080/15384101.2015.1026518?url_ver=Z39.88-2003&rfr_id=ori:rid:crossref.org&rfr_dat=cr_pub%3dpubmed">Abstract</a> and <a href="PDF Files/ACAP2.pdf">PDF</a>).</p>
    <p>&nbsp;</p>
    <p>Lin, J.L.*, Nakagawa, A.*, Skeen-Gaar, R.R., Yang, W.Z., Zhao, P., Zhang, Z., Ge, X., Mitani, S., Xue, D.#, and Yuan, H.S.# (2016). Oxidative Stress Impairs Cell Death by Repressing the Nuclease Activity of Mitochondrial Endonuclease G. <strong>Cell Reports </strong>16, 279–287. *Equal contribution. #Co-corresponding authors. (<a href="http://www.cell.com/cell-reports/fulltext/S2211-1247(16)30703-3">Abstract</a> and <a href="PDF Files/CPS-6 oxidized.pdf">PDF)</a>.</p>  
    <p></p>
  </p>
</article>
  <footer id="footer">
  <p data-gwd-node="true" data-gwd-uuid="yb2l2qbzlazc"> </p>
  &copy; 2014 University of Colorado Boulder-Xue Lab
  <p data-gwd-node="true" data-gwd-uuid="yb2l2qbzlazc"> </p>
</footer>
</div>
</body>
</html>
